__bass.py

```diff
@@ -24,11 +24,13 @@ def gen_script():
                        .check_output(['bash', '-c', command])
                        .split(divider, 1))
     new_env = new_env.lstrip().splitlines()
-
+    
+    new_env = [line for line in new_env if '{' not in line and '}' not in line]
+    
     old_env = dict([line.split('=', 1) for line in old_env])
     new_env = dict([line.split('=', 1) for line in new_env])
 
-    skips = ['PS1', 'SHLVL', 'XPC_SERVICE_NAME']
+    skips = ['PS1', 'SHLVL', 'XPC_SERVICE_NAME', 'PATH']
```

Comment:
Because the PATH syntax is different between bash and fish. We could include it if we add special handling for PATH to replace the : with a space.